### ISSUE TITLE
Fix inability to draft patterns

### ISSUE THREAD

**Issue by ghost** [joostdecock/site#402]
After the latest update I've been completely unable to draft new patterns. I suspect it has something to do with the 'theme'. When drafting a new pattern, neither Classic nor Paperless is selected. 

When explicitly selecting Paperless, there is just a "broken" picture where the draft normally is. When explicitly selecting Classic, an error pops up. 

Redrafting already existing drafts seems to work fine.

**Comment by ghost** [comment]
<img width="441" alt="screen shot 2018-05-25 at 15 21 21" src="https://user-images.githubusercontent.com/6172940/40546398-75bb8778-602f-11e8-8a1f-3411af89d760.png">
<img width="703" alt="screen shot 2018-05-25 at 15 24 07" src="https://user-images.githubusercontent.com/6172940/40546547-da40d874-602f-11e8-9479-897524902386.png">
<img width="423" alt="screen shot 2018-05-25 at 15 26 30" src="https://user-images.githubusercontent.com/6172940/40546633-0e57eb7a-6030-11e8-8110-0ae54cefd7fc.png">

**Comment by joostdecock** [comment]
Thank you for reporting this. When we migrated websites, we also migrated our backend to a different server. The migration script that synchronised the data did not set the permissions correctly, so while the draft was correctly created, it could not be saved to disk.

I've fixed the permissions now, and it should be fine. That being said, you are correct that there's something wrong with the theme settings, I'll create an new issue for that.
